pxtsim/logs.ts

```diff
@@ -335,14 +360,15 @@ namespace pxsim.logs {
 
                 if (entry.countElement) entry.countElement.innerText = entry.count.toString();
                 if (entry.accvaluesElement) entry.accvaluesElement.innerText = entry.value;
-                if (entry.chartElement) entry.chartElement.render();
+//                if (entry.chartElement) entry.chartElement.render();
+                if (entry.chartElement) new CanvasChart().drawChart(entry.chartElement, entry.accvalues.map(accvalue => new Point(accvalue.t, accvalue.v)))
```

Comment:
update canvaschart to take accvalues as input instead of allocating new point objects.

Reply:
Done